pkgs/by-name/ri/rimgo/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+  tailwindcss,
+}:
+buildGoModule rec {
+  pname = "rimgo";
+  version = "1.2.0";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "video-prize-ranch";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-C878ABs978viVtIuv3fPn2F2anOg2GB/+f5jaCO13tc=";
+  };
+  preBuild = ''
+    ${tailwindcss}/bin/tailwindcss -i static/tailwind.css -o static/app.css -m
+  '';
+  vendorHash = "sha256-u5N7aI9RIQ3EmiyHv0qhMcKkvmpp+5G7xbzdQcbhybs=";
+  meta = with lib; {
+    description = ''
+      An alternative frontend for Imgur
+    '';
+    platforms = platforms.unix;
+    homepage = "https://codeberg.org/video-prize-ranch/rimgo";
```

```suggestion
    homepage = "https://codeberg.org/rimgo/rimgo";
```
The repository has been moved (this should also be changed in `fetchFromGitea` above).